### ISSUE TITLE
404 error for main download link

### ISSUE THREAD

**Issue by Xansta** [daid/EmptyEpsilon#592]
Clicking on the download list link from the main page gives a 404 error. I know this is not strictly a build issue, but an issue with the site. Still, there was at least on potential player that could not get the download because the site was down.

**Comment by Jagernaut** [comment]
+ 6 Players

**Comment by oznogon** [comment]
http://software.ultimaker.com/ee_files.php doesn't exist

**Comment by daid** [comment]
Yes, Ultimaker asked me to host the files somewhere else. I still need to move it.

**Comment by amir-arad** [comment]
@daid need help with that? can you share the numbers (bandwith, storage size)?

**Comment by daid** [comment]
No idea on bandwidth requirements. I'm thinking about moving the releases to github releases system...

**Comment by blandin** [comment]
There are still problems with this.

The XHR on the GitHub Pages site for the files listing only functions over HTTP, so anyone browsing with an extension like HTTPS Everywhere--or even [simply loading the site over HTTPS](https://daid.github.io/EmptyEpsilon/)--isn't getting a list of download links on the downloads tab.

This wouldn't be more than an annoyance, however, if the [link to the download directory](http://daid.eu/ee/downloads/?C=M;O=D) wasn't also now throwing a 403.

**Comment by daid** [comment]
Download file listing is enabled again, as workaround. I might switch to github releases to manage this at some point.
